README.md

```diff
@@ -1,16 +1,58 @@
+  ```
+  $ gcloud kms keys versions get-public-key 1 --key snapshot --keyring sigstore-root --location global --project project-rekor | openssl ec -pubin -noout -text 
+  ```
+  * An online [timestamping key](https://github.com/sigstore/root-signing/blob/57ac5cd83b90ff97af78db846eea2525eb0eee81/repository/repository/root.json#L32-L42) located at `projects/project-rekor/locations/global/keyRings/sigstore-root/cryptoKeys/timestamp`
+   ```
+  $ gcloud kms keys versions get-public-key 1 --key timestamp --keyring sigstore-root --location global --project project-rekor | openssl ec -pubin -noout -text 
+  ```
+* [targets.json](repository/repository/targets.json): This is the list of trusted `targets.json` endorsed by the 5 root keyholders. It includes:
+  * [fulcio_v1.crt.pem](repository/repository/targets/artifact.pub): This is the [Fulcio](https://github.com/sigstore/fulcio) root certificate used to issue short-lived code signing certs. It is hosted at `https://fulcio.sigstore.dev`. You can `curl` the running root CA to ensure it matches the TUF root using `curl -v https://fulcio.sigstore.dev/api/v1/rootCert`
+  * [fulcio.crt.pem](repository/repository/targets/artifact.pub): This is the 
+  * [rekor.pub](repository/repository/targets/artifact.pub): This is the [Rekor](https://github.com/sigstore/rekor) public key used to sign entries and the tree head of the transparency log. You can retrieve the public key to ensure it matches with `curl -H 'Content-Type: application/x-pem-file' https://rekor.sigstore.dev/api/v1/log/publicKey`.
+  * [rekor.0.pub](repository/repository/targets/artifact.pub): This is a dupe of `rekor.pub` and will be removed in the next root-signing event.
+  * [ctfe.pub](repository/repository/targets/artifact.pub): Certificate Transparency log key that is used for certificates issued by Fulcio.
+  * [artifact.pub](repository/repository/targets/artifact.pub): Key that signs Cosign releases.
+* [snapshot.json]((repository/repository/snapshot.json)): This snapshot the valid metadata files. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
+* [timestamp.json]((repository/repository/timestamp.json)): The timestamp refreshes the most accurate snapshot file. It has a lifetime of 2 weeks and is resigned by a [GitHub workflow](https://github.com/sigstore/root-signing/blob/main/.github/workflows/snapshot-timestamp.yml).
+
+
+### Root locations
+
+The current root is published on a GCS bucket located at `https://storage.googleapis.com/sigstore-tuf-root`.
+
+
+## Current Sigstore Root Keyholders 
+* Bob Callaway [June 2021 - present]:
+  - TUF Key ID `f505595165a177a41750a8e864ed1719b1edfccd5a426fd2c0ffda33ce7ff209`: Yubikey material located [here](https://github.com/sigstore/root-signing/tree/main/ceremony/2021-06-18/keys/15938791).
```

Comment:
update to latest ceremony?

Reply:
Their key addition was performed in this ceremony though, so the blame history for the key is in that location.

Reply:
That's fair!